src/coreComponents/constitutiveDrivers/fluid/multiFluid/LogLevelsInfo.hpp

```diff
@@ -0,0 +1,58 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2016-2024 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2024 TotalEnergies
+ * Copyright (c) 2018-2024 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2023-2024 Chevron
+ * Copyright (c) 2019-     GEOS/GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file LogLevelsInfo.hpp
+ * This file contains common log level informations for physics solvers
```

Comment:
```suggestion
 * This file contains common log level informations for physics solvers
```


this does not seem to be for physics solvers but for constitutive models

Reply:
yes and there are some of these at other location